packages/babel-plugin-transform-object-rest-spread/src/shouldStoreRHSInTemporaryVariable.ts

```diff
@@ -1,13 +1,5 @@
-import { types as t } from "@babel/core";
+import type { types as t } from "@babel/core";
 
-const {
-  isObjectProperty,
-  isArrayPattern,
-  isObjectPattern,
-  isAssignmentPattern,
-  isRestElement,
-  isIdentifier,
-} = t;
```

Comment:
Accessing properties from `t` eagerly doesn't always work because of the CJS proxy. In theory it's fine, since `@babel/core` will load this plugin after that `@babel/core` is already loaded (so in real-world cases this wouldn't fail), but it was failing in our tests and just changing the code was easier.